README.pt.md

```diff
@@ -1,197 +1,197 @@
+MariaDB is the GPL version of MySQL.
+In debian: `sudo apt install mariadb-server`;
 
-#### Alterar usuário e senha
+#### Change username and password
 
-Alterar o arquivo de criação do banco de dados localizado em *doc/mysql.sql*:
+Change the database creation file located at * doc / mysql.sql * :
 
     CREATE DATABASE goworking DEFAULT CHARACTER SET utf8 DEFAULT COLLATE utf8_bin;
-    GRANT ALL PRIVILEGES ON goworking.* to usuario@'localhost' IDENTIFIED BY 'senha';
+    GRANT ALL PRIVILEGES ON goworking. * To user @ 'localhost' IDENTIFIED BY 'password';
 
-Mudar "usuario" para algum nome de usuário e "senha" para alguma senha.
+Change "user" to some username and "password" to any password.
 
-#### Rodar script de criação
+#### Run creation script
 
-Acessar o mariadb-client, copiar e colar os comandos ou rodar o script.
+Access mariadb-client, copy and paste the commands or run the script.
 
-No debian: `sudo mysql`
+In debian: `sudo mysql`
 
-    MariaDB [(none)]> SOURCE doc/mysql.sql;
+    MariaDB [(none)]> SOURCE doc / mysql.sql;
     Query OK, 1 row affected (0.001 sec)
-    
+
     Query OK, 0 rows affected (0.001 sec)
-    
+
     MariaDB [(none)]> SHOW DATABASES;
-    +--------------------+
-    | Database           |
-    +--------------------+
-    | goworking          |
+    + -------------------- +
+    | Database |
+    + -------------------- +
+    | goworking |
     | information_schema |
-    | mysql              |
+    | mysql |
     | performance_schema |
-    +--------------------+
+    + -------------------- +
     4 rows in set (0.001 sec)
 
-Se o que tiver disponível for um PHPMyAdmin ou coisa parecida, descobrir como é 
-o jeito difícil de colar estes comandos simples nesse sistema complicado.  
+If what you have available is a PHPMyAdmin or something, find out how it is
+the hard way to paste these simple commands into this complicated system.
 
 #### flask-migrate
 
-    user@server:goworking-mesas$ pipenv run flask db init
-    Loading .env environment variables...
-    user@server:goworking-mesas$ pipenv run flask db upgrade
-    Loading .env environment variables...
-    INFO  [alembic.runtime.migration] Context impl MySQLImpl.
-    INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
-    INFO  [alembic.runtime.migration] Running upgrade  -> fead50b08d21, empty message
-    INFO  [alembic.runtime.migration] Running upgrade fead50b08d21 -> dc7d560eee94, empty message
+    user @ server: goworking-tables $ pipenv run flask db init
+    Loading .env environment variables ...
+    user @ server: goworking-tables $ pipenv run flask db upgrade
+    Loading .env environment variables ...
+    INFO [alembic.runtime.migration] Context impl MySQLImpl.
+    INFO [alembic.runtime.migration] Will takes over non-transactional DDL.
+    INFO [alembic.runtime.migration] Running upgrade -> fead50b08d21, empty message
+    INFO [alembic.runtime.migration] Running upgrade fead50b08d21 -> dc7d560eee94, empty message
```

Comment:
These are standard error messages from the shell and shall not be reformatted / translated.